public/app/plugins/datasource/kairosdb/queryCtrl.js

```diff
@@ -69,7 +69,7 @@ function (angular, _) {
         }
       }
 
-      $scope.datasource.performTagSuggestQuery($scope.target.metric).then(function(result) {
+      $scope.datasource._performTagSuggestQuery($scope.target.metric).then(function(result) {
```

Comment:
this looks wrong, the controller should not access a private function (the leading underscore), should this not be something like metricFindQuery('tag_names(` + $scope.target.metric + ')')